doc/logo.md

```diff
@@ -0,0 +1,177 @@
+# Logo
+
+"I too see the computer presence as a potent influence on the human mind. 
+ I am very much aware of the holding power of an interactive computer and 
+ of how taking the computer as a model can influence the way we think about 
+ ourselves. In fact the work on LOGO to which I have devoted much of the past
+ years consists precisely of developing such forces in positive directions."
+
+Seymour Papert
+
+"Logo is the name for a philosophy of education and for a continually 
+ evolving family of computer languages that aid its realization."
+
+Harold Abelson
+
+"Historically, this idea that Logo is mainly turtle graphics is a mistake. 
+Logo’s name comes from the Greek word for word, because Logo was first
+designed as a language in which to manipulate language: words and sentences."
+
+Brian Harvey
+
+Logo was initially created by Wally Feurzeig, Seymour Papert, and Cynthia
+Solomon in 1967 as part of National Science Foundation sponsored research
+project conducted at Bolt, Beranek & Newman, Inc., in Cambridge, Massachusetts.
+In 1969 Dr. Seymour Papert started the Logo Group at the MIT Artificial Intelligence
+Lab. Throughout the 1970s the majority of Logo development was conducted at MIT
+in the Artificial Intelligence Lab and the Division for Study and Research in
+Education, using large research computer systems, such as ITS powered PDP-10.
+
+Our goal is to make that early Logo systems available to wider auditory of
+enthusiasts for exploration, experimenting and, of course, hacking.
+
+### BBN Logo
+
+BBN PDP-10 LOGO system was implemented originally by Walter B. Weiner with
+the assistance of Paul Wexelblat and Charles R. Morgan at BBN. 
+We have source file with a copyright from 1970. It is version modified at National
+Research Council, Ottawa, Canada by A.G. Smith and R.A.Orchard.
+That version of BBN Logo for PDP-10 TOPS-10 and TENEX was assembled with MACRO-10
+in ITS, and dumped with the DECUUO bootstrap to get ITS executable binary by Lars
+Brinkhoff. 
+
+to run, type:
+`:bbn;logo`
+
+Here is some BBN Logo related documentation:
+
+ Bolt Beranek and Newman, Inc., Report no. 2165 vol. 4 "The Logo Processor: A Gide 
+ for System Programmers", pub date 30 June 1971.
+
+ Bolt Beranek and Newman, Inc., Report no.1889 "Programming-Languages as a Conceptual 
+ Framework for Teaching Mathematics" (Feurzeig, Papert at al., 1969)
+ Appendix "A Description of the Logo Language and System".
+
+ Both documents are available at:
+
+ https://github.com/PDP-10/bbn-logo
+ 
+### MIT CLOGO
+
+MIT CLOGO is a direct BBN LOGO descendant. It is, in essence, BBN Logo ported to ITS
+and MIDAS and enhanced with turtle graphics and other capabilities at MIT.
+
+We have pretty stable CLOGO version 49 binary as well as good-commented source code for
+the version.
```

Comment:
"We have **a** pretty stable..."  
"*well*-commented source code"

Reply:
done.